sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.projectKey=mrbeam_MrBeamPlugin
+sonar.organization=mrbeam
+
+# This is the name and version displayed in the SonarCloud UI.
+sonar.projectName=MrBeamPlugin
+sonar.projectVersion=v0.11.0b3
```

Comment:
Not sure about this one. May be it needs to be revised later.
Here I just put the latest release

Reply:
I can't see the version in the sonar cloud UI

Reply:
me neither . needs to be investigated more